test-cypress/stubs/cluster/cluster-stubs.js

```diff
@@ -191,4 +183,60 @@ export class ClusterStubs extends Stubs {
         }).as('2-nodes-cluster-created');
     }
 
+    static stubSaveClusterConfiguration(expectedRequestBody) {
+        cy.intercept({
+            method: 'PATCH',
+            url: '/rest/cluster/config'
+        }, (req) => {
+            req.reply({
```

Comment:
Why? I don't see anything that requires the request handler. Wouldn't a simple interceptor with a fixture work just fine in this case?

Reply:
added fixture